src/clevertap.js

```diff
@@ -567,7 +577,7 @@ export default class CleverTap {
   }
 
   // starts here
-  init (accountId, region, targetDomain) {
+  init (accountId, region, targetDomain, token) {
```

Comment:
I would prefer a default value here. Also we are duplicating code. We are checking whether token is passed in the account class as well. We do not need to

Reply:
Had a chat, this is fine